### PR TITLE
rename dir and pull in scripts for config editor

### PR DESCRIPTION
**Pull request by dcwebermn**

### CODE REVIEW

File: acd-config-editor/scripts/acd-ce-images.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# list of images to pull and/or load
```
Comment (cagrev) [inline acd-config-editor/scripts/acd-ce-images.txt:1]: These will need to be updated again soon, but fine for now.

**Review by cagrev — APPROVED** [review]
I didn't review the details of the text files (deploy, pull, readme, etc.) coming over. Assuming those were already set/reviewed when previously published. Once merged in, we can do a more complete review/test as needed.

**Review by herbeck558 — COMMENTED** [review]
We either need to keep the existing directory w/ helm charts where they are or update the EAP instructions to point to the new location - i'd just leave a copy where they were for now).
This new dir can have the source and maybe should be the master going forward but also needs to publish single .tar.gz files (a latest and one per old release going forward) to make it easy to download just that w/o cloning or downloading this entire repo.  We can generate the new .tar.gz via jenkins and it should have these scripts/text files along w/ the charts all in one.

Also the readme in the internal repo should be removed from this public repo - the instructions are going into the gh pages (currently in debs branch).

File: acd-config-editor/docker_deploy_service.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Watson Health Cognitive Services (WHCS) Docker Instructions
```
Comment (herbeck558) [inline acd-config-editor/docker_deploy_service.md:1]: Remove this file - in the pages source that deb has will have the docker install instructions.  we can have a README.md in this new folder that points to that doc on the published pages.

File: acd-config-editor/scripts/readme.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Watson Annotator for Clinical Data Configuration Editor stand-alone installation Instructions
```
Comment (herbeck558) [inline acd-config-editor/scripts/readme.md:1]: remove this file  -its out of date and for internal how we built the old .tar file for the eap program now anything for clients.

**Pull request closed without merging**